package.json

```diff
@@ -80,21 +80,22 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "pnpm test:development:web-test-runner",
-    "test:development:playwright": "playwright test --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
+    "test:development:playwright": "http-server ./dist/playwright/coverage --silent & playwright test --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
```

Comment:
`http-server` is back! 

Playwright is able to serve coverage reports. But it would open a browser tab with the report every time this script is run. If you're okay with that, let me know. I'll replace `http-server` with the Playwright command.